src/btest.cr

```diff
@@ -329,73 +281,56 @@ class Case
 
     if validation_errors.size > 0
       return Result.new(runner, self, false,
-                        validation_errors +
-                        "\nstdout: #{stdout}\nstderr: #{stderr}")
+        validation_errors + "\nstdout: #{stdout}\nstderr: #{stderr}")
     end
 
     Result.new(runner, self, true, "")
   end
 end
 
-
 # This is the result of a test case run with a particular runner.
 class Result
-  def initialize(runner : Runner, testCase : Case, pass : Bool,
-                 message : String)
-    @runner = runner
-    @testCase = testCase
-    @pass = pass
-    @message = message
-  end
-
-  def testCase
-    @testCase
-  end
+  getter testCase, pass, message
 
-  def pass
-    @pass
-  end
-
-  def message
-    @message
+  def initialize(@runner : Runner, @testCase : Case, @pass : Bool,
+                 @message : String)
   end
 
   def render
     name = @testCase.name.as(String)
 
     uncolored_len = @testCase.suite.name.size + 1 + @runner.name.size + 1 + \
-                    name.size
+      name.size
 
-    # Chop the output if the name is too long
-    chop_delta = uncolored_len - (TERMINAL_WIDTH - 7)
-    if chop_delta > 0
-      name = name[0..name.size - chop_delta]
-      uncolored_len -= chop_delta - 1
-    end
+  # Chop the output if the name is too long
```

Comment:
Why is there indentation changes here? (maybe a bug in the formatter?)

Reply:
Yeah, looks like a bug :sweat_smile:

Reply:
Yeah, is a bug:

Minimal example:

```crystal
def foo
  bar = ""
  bar = "foo" + \
    "bar"
bar
end
``` 

I guess is fixed on master, no? see: https://github.com/crystal-lang/crystal/issues/5892#issuecomment-377741557